lms/templates/seq_module.html

```diff
@@ -1,40 +1,36 @@
 <%! from django.utils.translation import ugettext as _ %>
 
 <div id="sequence_${element_id}" class="sequence" data-id="${item_id}" data-position="${position}" data-ajax-url="${ajax_url}" >
-  <nav class="sequence-nav">
-    <ul class="sequence-nav-buttons">
-      <li class="prev"><a role="button" href="#">${_('Previous')}</a></li>
-    </ul>
-
-    <div class="sequence-list-wrapper">
-      <ol role="tablist" aria-label="${_('Section Navigation')}" id="sequence-list">
-        % for idx, item in enumerate(items):
-        ## TODO (vshnayder): add item.progress_detail either to the title or somewhere else.
-        ## Make sure it gets updated after ajax calls.
-        ## implementation note: will need to figure out how to handle combining detail
-        ## statuses of multiple modules in js.
-        <li>
-        <a class="seq_${item['type']} inactive progress-${item['progress_status']}"
-           data-id="${item['id']}"
-           data-element="${idx+1}"
-           href="javascript:void(0);"
-           title="${item['title']|h}"
-           data-page-title="${item['page_title']|h}"
-           aria-controls="seq_contents_${idx}"
-           id="tab_${idx}"
-           tabindex="0"
-           role="tab">
-            <p aria-hidden="false">${item['title']}</p>
-          </a>
-        </li>
-        % endfor
-      </ol>
-    </div>
-
-    <ul class="sequence-nav-buttons">
-      <li class="next"><a role="button" href="#">${_("Next")}</a></li>
-    </ul>
-  </nav>
+  <div class="sequence-nav">
+    <nav class="sequence-list-wrapper" aria-label="Unit">
+      <button class="sequence-nav-button button-previous prev">${_('Previous')}</button>
+      <div class="sequence-list">
+        <ol aria-label="${_('Section Navigation')}" id="sequence-list">
+          % for idx, item in enumerate(items):
+          ## TODO (vshnayder): add item.progress_detail either to the title or somewhere else.
+          ## Make sure it gets updated after ajax calls.
+          ## implementation note: will need to figure out how to handle combining detail
+          ## statuses of multiple modules in js.
+          <li>
+            <a class="seq_${item['type']} inactive progress-${item['progress_status']}"
+             data-id="${item['id']}"
+             data-element="${idx+1}"
+             href="javascript:void(0);"
+             title="${item['title']|h}"
+             aria-label="${item['title']|h}"
+             data-page-title="${item['page_title']|h}"
+             aria-controls="seq_contents_${idx}"
+             id="tab_${idx}"
+             tabindex="0">
+              <p aria-hidden="false">${item['title']}</p>
+            </a>
+          </li>
+          % endfor
+        </ol>
+      </div>
+      <button class="sequence-nav-button button-next next">${_('Next')}</button>
```

Comment:
Is the ``next`` class redundant now that you've added the classes ``sequence-nav-button``and ``button-next``? Can we remove it from the DOM, styling, and any JS/test logic?